example-apps/tests-cucumber/exoservice/src/server.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+
+  beforeAll: (done) => {
+    // TODO: add asynchronous init code here, or delete the whole block
```

Comment:
please remove this comment and the beforeAll as a whole since we aren't using it